Missionframework/scripts/server/init_server.sqf

```diff
@@ -126,3 +126,26 @@ execVM "scripts\server\offloading\group_diag.sqf";
 if (KP_liberation_restart > 0) then {
     execVM "scripts\server\game\server_restart.sqf";
 };
+
+if (KP_liberation_limited_zeus) then {
+    zm1 setVariable ["Addons", 0, true];
+    removeAllCuratorAddons zm1;
+
+    zm1 setCuratorCoef ["edit", -1e8];
+    zm1 setCuratorCoef ["place", -1e8];
+    zm1 setCuratorCoef ["synchronize", 0];
+    zm1 setCuratorCoef ["delete", 0];
```

Comment:
Not sure about this (delete), it was always set to zero but it seems like an oversight to me. Limited curator should not be allowed to destroy objects IMO. Synchronize needs to stay as it allows to group up units.

Reply:
Yeah, that s why I think it's an oversight, however it was like that for years and no one complained 🤔

Reply:
It's needed for being able to "recycle" the helipads, which can't be targeted in normal view for displaying an action menu entry. And to maybe delete own AI units etc. if they aren't needed anymore, far far away or the AI got stuck etc.
Concerning enemy units: These aren't added to Zeus and therefore won't be able to be deleted by the commander.